idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/TestShuffler.java

```diff
@@ -94,15 +95,15 @@ private String historicalType() {
             return reverseOrder();
         }
 
-        final Path historicalRun = DetectorPathManager.detectionRoundPath(historicalType(), i);
+        /*final Path historicalRun = DetectorPathManager.detectionRoundPath(historicalType(), i);
```

Comment:
While I recognize we don't seem to actually use this logic, maybe for this PR we should just keep it as is (after any changes necessary for the conversion), just to keep the PR something that preserves functionality but with the new API.

Reply:
In other words, you would like me to uncomment out this block? Despite potential conflicts with returning a different value for the function?

Reply:
What would have to change? Would we have to pass some baseDir through as well here to get things to work?

Reply:
Go thru and find instantiations of TestShuffler, change it so that it passes in a baseDir, change to 1st param of the commented out block